core/controlplane/config/templates/cloud-config-controller

```diff
@@ -500,6 +500,11 @@ write_files:
           /usr/bin/docker run --rm --net=host -v /srv/kubernetes:/srv/kubernetes {{.HyperkubeImage.RepoWithTag}} /hyperkube kubectl "$@"
       }
 
+      while ! kubectl get ns kube-system; do
```

Comment:
Not sure 😢 
Neither kubernetes nor coreos-overlay(wrapping kubelet) seems to create it.
Very unlikely, but would it be implicitly created via static pods in `/etc/kubernetes/manifests`?